tests/test_filter.py

```diff
@@ -0,0 +1,311 @@
+"""
+Unit test fro imunopepper/dna_to_peptide.pyx.
+"""
+import numpy as np
+import pytest
+import scipy as sp
```

Comment:
I think scipy import is not needed here. I checked and it is only used for `sp.array`, which is the same as `np.array`. Would only use the latter.